postcss.config.js

```diff
@@ -0,0 +1,12 @@
+const postcssPresetEnv = require('postcss-preset-env');
```

Comment:
why is this the only variable being explicitly defined like this?

Reply:
Resolved